code/modules/research/machinery/_production.dm

```diff
@@ -339,12 +333,12 @@
 	if(!items_remaining)
 		finalize_build()
 		return
-	addtimer(CALLBACK(src, PROC_REF(do_make_item), design, materials_per_item, time_per_item, items_remaining), time_per_item)
+	addtimer(CALLBACK(src, PROC_REF(do_make_item), design, build_efficiency, time_per_item, items_remaining), time_per_item)
 
 /// Resets the busy flag
 /// Called at the end of do_make_item's timer loop
 /obj/machinery/rnd/production/proc/finalize_build()
-	PROTECTED_PROC(TRUE)
+	PRIVATE_PROC(TRUE)
```

Comment:
these were made protected for a reason

Reply:
But why though? Doesn't look like it was used anywhere else

Reply:
downstreams exist.

Reply:
Ahhh